jetson/include/core/constants.h

```diff
@@ -12,12 +14,16 @@ const float FLOOR_2_CAM = 0.890;        // [m]
 const float CAMERA_2_EFFECTOR_X = 0.032;    // [m]
 const float CAMERA_2_EFFECTOR_Y = 0.058;    // [m]
 
-const float DRIVE_ENC_2_DIST = 999; // [m/count]
-const float XY_ENC_2_DIST = 999;    // [m/count]
+// (hopefully not painfully) slow to start
+const float IDEAL_SPEED_DRIVE = 0.1;    // [m/s]
+const float IDEAL_SPEED_ARM = 0.05;     // [m/s]
 
-const float WHEEL_RADIUS = 0.0762; // [m]
+const float WHEEL_RADIUS = 0.0762;                              // [m]
+const float DRIVE_ENC_2_DIST = 2*M_PI/8*2*M_PI*WHEEL_RADIUS;    // [m/count]
```

Comment:
This seems wrong. Shouldn't the conversion be `(wheel circumference)/n_enc` = `(2 * M_PI * WHEEL_RADIUS) / 8`

Reply:
yeah i think arc length = (2 * M_PI * WHEEL_RADIUS) / 8